src/commands/sbom/types.ts

```diff
@@ -108,5 +108,6 @@ export interface ScaRequest {
   dependencies: Dependency[]
   service: string
   env: string
+  source: 'ci'
```

Comment:
There is already the source of the scan identifier in the header of the request to the API. This change should not be done in the API but rather sent by CWP. See [this file](https://github.com/DataDog/dd-source/blob/main/domains/static-analysis/apps/apis/static-analysis-sca-api/internal/services/evp_sbom.go#L21).

Reply:
this is great info, just what I needed, thanks!